my-project/src/Components/Exercise6.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+function GreetForm() {
+  // Initialize state variables for first name and last name
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
```

Comment:
Using a single state variable for 'firstName' and 'lastName' might be more efficient since they're used together in some places. You can try this:
```suggestion
  const [name, setName] = useState({ firstName: '', lastName: '' });
```